Dockerfile

```diff
@@ -27,8 +27,19 @@ RUN apt-get update -y && \
     rm -rf /var/lib/apt/lists/*
 
 # Serve frontend application
-FROM docker.io/library/nginx:1.19
+FROM docker.io/library/nginx:1.25
 COPY --from=react-build /code/reana-ui/build /usr/share/nginx/html
 COPY nginx/reana-ui.conf /etc/nginx/conf.d/default.conf
 EXPOSE 80
 CMD ["nginx", "-g", "daemon off;"]
+
+LABEL org.opencontainers.image.authors="team@reanahub.io"
+LABEL org.opencontainers.image.created="2023-12-12"
+LABEL org.opencontainers.image.description="REANA UI"
+LABEL org.opencontainers.image.documentation="https://reana-ui.readthedocs.io/"
```

Comment:
Since each paragraph in `Dockerfile` is accompanied by a leading comment, you can precede the label setting section by:

```
# Set image labels
```